### PR TITLE
Add Project Proposal

### PR DESCRIPTION
**Pull request by Vid201**

### CODE REVIEW

File: projects/4337-bundler-rust.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+- JSON-RPC API: 2 weeks.
+    - Implement all JSON-RPC API endpoints and necessary logic.
+- User operation pool: 5 weeks.
+    - First, implement a standalone mempool with gRP endpoints and all the necessary logic, such as simulation and validation of user operations.
+    - Work on p2p mempool and connect with other bundlers (this requires p2p connections to other clients - will look into reusing Akula p2p interfaces and code).
+- Bundler: 3 weeks.
+    - Implement all logic for bundling - simulation, signature aggregation, bundle construction, and propagating bundles as transactions to the txpool.
+    - First, start with public txpool, then add Flasbots Protect (or send directly to the block producer over gRPC).
+- Others: 1 week (and later).
+    - Research and sketch out necessary work to prototype integration into Akula.
```
Comment (zsluedem) [inline projects/4337-bundler-rust.md:38]: Is it possible to integrate with some other clients?

File: projects/4337-bundler-rust.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    - Other advanced features.
```
Comment (zsluedem) [inline projects/4337-bundler-rust.md:39]: :+1:

**Review by zsluedem — APPROVED** [review]
Good work!

File: projects/4337-bundler-rust.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+1. Demonstrate successful implementation by connecting and using one of the EIP-4337 compatible wallets.
+1. Contribute to official bundler specs and pass the test suite.
+1. Outline the plan and work to push the implementation into Akula's main codebase.
+
+## Collaborators
+
+### Fellows 
+
+- [Vid Kersic](https://github.com/Vid201)
+- [WillQ](https://github.com/zsluedem) (in the conversation, not fully decided yet)
```
[the remark this answers is not in the document]
Comment (Vid201) [reply projects/4337-bundler-rust.md:63]: Ofc :)

**Pull request merged**